crates/eframe/src/native/glow_integration.rs

```diff
@@ -423,15 +424,6 @@ impl WinitApp for GlowWinitApp {
                 EventResult::Wait
             }
 
-            winit::event::Event::MainEventsCleared => {
-                if let Some(running) = &self.running {
-                    if let Err(err) = running.glutin.borrow_mut().on_resume(event_loop) {
-                        log::warn!("on_resume failed {err}");
-                    }
-                }
-                EventResult::Wait
-            }
-
```

Comment:
We should be able to remove this, as  `winit::event::Event::Resumed` is handled above.

Reply:
While you're working on `glow_integration`, it's probably wise to remove this stale comment entirely:

https://github.com/emilk/egui/blob/e037489ac20a9e419715ae75d205a8baa117c3cf/crates/eframe/src/native/glow_integration.rs#L36-L46

The issue linked there has long been closed and fixed. Windows/Surfaces are _only associated with context **when/while it's made current**_, and it looks like `eframe` supports Android now with a proper `Resumed`/`Suspended` cycle?